src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py

```diff
@@ -145,6 +145,11 @@ def body(self):
         self.assertTrue('s2' not in result)
         self.assertTrue('s3' in result)
 
+        self.cmd('appservice web config hostname list -g {} --webapp {}'.format(self.resource_group, self.webapp_name), checks=[
```

Comment:
the `add` and `delete` were manually verified, as it involves manually setup the CNAME records in my domain provider's portal